app/backend/app.py

```diff
@@ -289,6 +302,23 @@ async def chat(request: Request):
         log.error(f"Error in chat:: {ex}")
         raise HTTPException(status_code=500, detail=str(ex)) from ex
 
+@app.get("/getstreamlitip")
+async def get_streamlit_ip():
```

Comment:
same - should this be removed if we are no longer using it?

Reply:
removed.